content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-12.md

```diff
@@ -0,0 +1,289 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Upgrading kubeadm clusters from v1.11 to v1.12
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains how to upgrade a Kubernetes cluster created with `kubeadm` from version 1.11.x to version 1.12.x, and from version 1.12.x to 1.12.y, where `y > x`.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+- You need to have a `kubeadm` Kubernetes cluster running version 1.11.0 or later. Swap must be disabled. The cluster should use a static control plane and etcd pods.
+- Make sure you read the [release notes](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.12.md) carefully.
```

Comment:
s/1.12 release notes/release notes would be better?

Reply:
This upgrade doc might be used after 1.13 comes out, so I think this makes sense.

Reply:
I was thinking of that since the link is for 1.12 so it'd be good to show the version in the text. It's okay as-is, anyway :)